modules/sigprofiler/1.0/src/python/run_extractor.py

```diff
@@ -24,17 +25,17 @@ def main():
 
 def parse_arguments():
     parser = argparse.ArgumentParser()
-    parser.add_argument('input_data')
-    parser.add_argument('output')
-    parser.add_argument('ref')
-    parser.add_argument('context_type', default = '96,DINUC,ID')
-    parser.add_argument('exome', default = False, type = bool)
-    parser.add_argument('minimum_signatures', default = 1, type = int)
-    parser.add_argument('maximum_signatures', default = 25, type = int)
-    parser.add_argument('nmf_replicates', default = 100 , type = int)
-    parser.add_argument('matrix_normalization', default = 'gmm')
-    parser.add_argument('nmf_init', default = 'nndsvd_min')
-    parser.add_argument('cpu', default = -1, type = int)
+    parser.add_argument('input_data', help = 'Path to matrix that will undergo NMF.')
```

Comment:
Have you set it up so these options are available to be tweaked by the user via the config?